async_fuse/src/protocol.rs

```diff
@@ -424,6 +424,7 @@ pub enum FuseNotifyCode {
 }
 
 // The read buffer is required to be at least 8k, but may be much larger
+#[allow(dead_code)]
```

Comment:
for unused const variables, add a mod to group them, and allow dead_code for the mod.

Reply:
done